examples/website-demos/social-messaging/src/components/ChatContainer/ChatContainer.tsx

```diff
@@ -94,18 +94,19 @@ export const ChatContainer: React.FC = () => {
     };
   }, []); // eslint-disable-line
 
-  const customRenderFilter = (channels: StreamChannel[]) => {
+  const CustomRenderFilter = (channels: StreamChannel[]) => {
```

Comment:
this should still be lowercase since it's not a component, but otherwise all looking good